<launch>

    <!--  Point to the map file created during mapping -->
    <arg name="map_file" default="$(find kinect_challenge)/maps/map.yaml"/>

   
    <include file="$(find kinect_challenge)/launch/view_robot.launch" />
  
    <node type="MSBMIOSvs.py" name = "MSBMIOSvs"    pkg="kinect_challenge" >
        <!-- host IP address, IP address of PC running Microsoft Benchmark -->
        <!-- set to "sim" to simulate connection to the Microsoft app -->
        <param name="host_IP" type="str" value="sim" />
        
        <!-- port to talk to Microsoft Benchmark on -->
        <param name="host_port" type="int" value="7576" />
    </node>
    
    <node type="Navigator.py" name = "Navigator"    pkg="kinect_challenge" >
        <param name="MapPath" type="str" value="$(find kinect_challenge)/maps" />
        
        <!-- Waypoints saved here, file in maps folder -->
        <param name="WaypointFile" type="str" value="waypoints.wp" />
        
        <!-- Create/Edit this file to determine the route to take during navigation
            file is in the maps folder.
            syntax: One landmark pair per line, landmark ids 0 based
            <to landmark id>  <from landmark id>
            ...
            
        -->
        <param name="WaypointTour" type="str" value="tour.wpt" />
    </node>
    
</launch>
